src/Proj4.jl

```diff
@@ -50,14 +48,21 @@ function _get_def(proj::Projection)
 end
 
 """Low level interface to libproj transform, allowing user to specify strides"""
-function _transform!(src::Projection, dest::Projection, point_count, point_stride, x, y, z)
+function _transform!(src::Projection, dest::Projection, point_count::Int64,
+                     point_stride::Int32, x::Ptr{Float64}, y::Ptr{Float64}, z::Ptr{Float64})
```

Comment:
You're right that I haven't been very consistent in sticking to the platform independent types